make fails on linux

```
➜  loki git:(promtail-ext-lbl) ✗ make
docker run --rm --tty -i \
	-v /home/goutham/go/src/github.com/grafana/loki/.cache:/go/cache \
	-v /home/goutham/go/src/github.com/grafana/loki/.pkg:/go/pkg \
	-v /home/goutham/go/src/github.com/grafana/loki:/go/src/github.com/grafana/loki \
	grafana/loki-build-image pkg/parser/labels.go;
make: Entering directory '/go/src/github.com/grafana/loki'
make: 'pkg/parser/labels.go' is up to date.
make: Leaving directory '/go/src/github.com/grafana/loki'
docker run --rm --tty -i \
	-v /home/goutham/go/src/github.com/grafana/loki/.cache:/go/cache \
	-v /home/goutham/go/src/github.com/grafana/loki/.pkg:/go/pkg \
	-v /home/goutham/go/src/github.com/grafana/loki:/go/src/github.com/grafana/loki \
	grafana/loki-build-image cmd/loki/loki;
make: Entering directory '/go/src/github.com/grafana/loki'
go build -ldflags "-extldflags \"-static\" -linkmode=external -s -w" -tags netgo -i -o cmd/loki/loki ./cmd/loki
go build net: open /usr/local/go/pkg/linux_amd64/net.a: permission denied
Makefile:102: recipe for target 'cmd/loki/loki' failed
make: *** [cmd/loki/loki] Error 1
make: Leaving directory '/go/src/github.com/grafana/loki'
make: *** [Makefile:93: cmd/loki/loki] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -78,7 +78,7 @@ RM := --rm
 # as it currently disallows TTY devices. This value needs to be overridden
 # in any custom cloudbuild.yaml files
 TTY := --tty
-GO_FLAGS := -ldflags "-extldflags \"-static\" -s -w" -tags netgo -i
+GO_FLAGS := -ldflags "-extldflags \"-static\" -s -w" -tags netgo
 NETGO_CHECK = @strings $@ | grep cgo_stub\\\.go >/dev/null || { \
        rm $@; \
        echo "\nYour go standard library was built without the 'netgo' build tag."; \

